spec/validation/configure_spec.rb

```diff
@@ -34,244 +41,264 @@
                'and node will cause an error.',
```

Comment:
Can we just make this a comment rather than having it split up across the value for several questions? This is slightly confusing, particularly as these aren't valid questions, and will also make the text harder to edit than if it was all in one comment. It may be clearest if the value for `questions` is just `{}` since we don't care about the value (I think?).

Also this comment is now out of date with the addition of `self` - could be worth changing it to say that the valid top keys are `questions` + the values in `Metalware::Repo::CONFIGURE_SECTIONS`.

Reply:
fixed in: https://github.com/alces-software/metalware/pull/185/commits/b9d2c4a5610a4499e95c29f8c62ab1ff023d2df9 https://github.com/alces-software/metalware/pull/185/commits/be52f5d7d3cd6efce621c6ad5fa836a1ee2b817a